Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -267,6 +267,28 @@ public async Task<MatchViewModel> UpdateMatchAsync(int matchID, MatchPatchViewMo
             return match;
         }
 
+        public IEnumerable<ParticipantAttendanceViewModel> GetMatchAttendance(int matchID)
+        {
+            var match = _context.Match.Find(matchID);
+            var res = new List<ParticipantAttendanceViewModel>();
+            if (match is null) return res;
+
+            foreach(MatchTeam mt in match.MatchTeam)
+            {
+                var attendance = _context.MatchParticipant
+                    .Where(mp => mp.TeamID == mt.TeamID && mp.MatchID == mt.TeamID)
```

Comment:
Here should be `mp.MatchID == mt.MatchID` right? Not `mt.TeamID`.

Reply:
good catch. I think it most likely auto filled and I didn't notice.